dbms/src/Interpreters/Aggregator.h

```diff
@@ -1454,13 +1453,27 @@ class Aggregator
         AggProcessInfo & agg_process_info,
         TiDB::TiDBCollators & collators) const;
 
-    template <bool collect_hit_rate, bool only_loopup, typename Method>
-    void executeImplBatch(
+    template <bool collect_hit_rate, bool only_loopup, bool enable_prefetch, typename Method>
+    void executeImplMiniBatch(
```

Comment:
Seems no `Mini` is better because the mini batch is used only when `enable_prefetch` is true.